src/components/Books.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+/* eslint-disable react/prop-types */
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/bookSlice';
+
+const CreateBook = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveClick = () => {
+    dispatch(removeBook({ id }));
+  };
 
-const Books = (props) => {
-  const { title, author } = props;
   return (
-    <div>
-      <ul>
-        <li>{title}</li>
-        <li>{author}</li>
-      </ul>
-      <button type="button">Remove</button>
+    <div className="books_wrapper">
+      <div className="books">
+        <p>{title}</p>
+        <p>{author}</p>
+      </div>
+      {title && (
+      <div className="buttons">
+        <button type="button">comments</button>
+        <button type="button" onClick={handleRemoveClick}>
```

Comment:
- [ ] While I am able to add book, when I click the remove button the book is not removed. Kindly fix that. Ensure that your action is being dispatched to the store correctly and the book is removed.